lib/tasks/gem_owners.rake

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+#======================================================================================
+# HOW TO USE THIS
+# This task assumes that you have rubygems loaded into your database, but no users
+# The task will create users and assign them to the most downloaded gems
+#
+# To clear your database and load the latest rubygems data dump:
+# 💻 In your rubygems.org directory run ... (optional)
+#    rails db:reset
+# 💻 Navigate to Rubygems Data Dump and download one of the dumps (https://rubygems.org/pages/data)
+# 💻 In your rubygems.org direction run ...
+#    ./script/load-pg-dump -d rubygems_development ~/Downloads/public_postgresql.tar
+# 💻 In your rubygems.org direction run ...
+#    rake gem_owners:seed
+#======================================================================================
+
+namespace :gem_owners do
+  task seed: :environment do
+    people = [
+      {
+        email: "sandi@metz.com",
```

Comment:
Please use generic domains for examples (testing) per https://www.iana.org/domains/reserved. In the end, it would be best to use user factories (https://github.com/rubygems/rubygems.org/blob/master/test/factories.rb#L10) already using `example.com`, create minimal amount of users and print the login/password combination to the console.

Also is there any reason to not use `db:seed` (or at least reuse the namespace -> make it `db:seed:owners`)? It could be possible to add users in `db/seed.rb` and if any gems are already preset, add ownership. On the otherside I'm not sure how useful is to add all gems to one user.